benchmark/resharding/README.md

```diff
@@ -0,0 +1,68 @@
+# Benchmark
```

Comment:
please move the `resharding` folder under `benchmark/alpa` folder(that is, `benchmark/alpa/resharding`)

Reply:
OK